pkg/services/sqlstore/migrations/ualert/tables.go

```diff
@@ -23,6 +23,8 @@ func AddTablesMigrations(mg *migrator.Migrator) {
 	AddAlertRuleMigrations(mg, 60)
 	AddAlertRuleVersionMigrations(mg)
 
+	AddAlertmanagerConfigHistoryMigrations(mg)
```

Comment:
This is the only safe place to put this. Even then, it's prone to bugs. See https://github.com/grafana/grafana/issues/60196